software/utils/rpi_config/bootstrap.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+
+# Fail if any command fails
+set -e
+set -o pipefail
+
+echo "==============================================================================="
+echo "=================== RespiraWorks Ventilator bootstrapper ======================"
+echo "==============================================================================="
+echo " "
+echo "  -- clones the repository"
+echo "  -- configures Raspberry Pi:"
+echo "     * no screensaver or screen blanking"
+echo "     * no splash on bootup"
+echo "     * serial interface enabled for GUI<->controller comms"
+echo "  -- udev rule added for nucleo, enabling flashing of firmware"
+echo "  -- guake console accessible at all times using [F12]"
+echo "  -- desktop shortcuts for ventilator GUI, updater, debug interface"
+echo "  -- RW-theme background"
+echo "  -- installs toolchains and dependencies for building controller and GUI"
+echo "  -- will require a restart when done"
+echo " "
+echo " Will require a restart when complete."
+read -n1 -s -r -p $'Press any key to continue...\n' key
+
+# Print each command as it executes
+set -o xtrace
+
+# Check if Linux
+PLATFORM="$(uname -s)"
+if [ $PLATFORM != "Linux" ]; then
+  echo "Error: This script only supports 'Linux'. You have $PLATFORM."
+  exit 1
+fi
+
+sudo apt-get update
+sudo apt-get -y upgrade
+sudo apt-get --yes install guake git-lfs
+
+### sudo apt-get --yes --allow-downgrades, --allow-remove-essential --allow-change-held-packages install guake git-lfs
```

Comment:
Should this stay here or is it commented out for good?

Reply:
I will remove this. I guess we don't need the `apt upgrade` either if we assume the user already did this as part of general raspi init.